login-workflow/docs/existing-project-migration.md

```diff
@@ -0,0 +1,135 @@
+
+Update @brightlayer-ui/react-auth-workflow package to latest version.
+```shell
+npm install --save @brightlayer-ui/react-auth-workflow
+// or
+yarn upgrade @brightlayer-ui/react-auth-workflow
+```
+
+## Remove @brightlayer-ui/react-auth-shared package
+
+In latest version we have removed @brightlayer-ui/react-auth-shared.
+
+## Update AuthUIActions
+
+Since we have removed `@brightlayer-ui/react-auth-shared` package `SecurityContextActions` are not supported by latest workflow. Instead of this you need to manage your app's state by app context. 
+
+For e.g. you can replace SecurityContextActions with `AppContextType` which you can declare in AppContextProvider. You can check [here](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/example/src/contexts/AppContextProvider.tsx) for detailed implementation.
+
+In previous package `onUserAuthenticated`, `onUserNotAuthenticated` were managed by `@brightlayer-ui/react-auth-shared` but in latest you need to manage through app state. For more details you can check Example app's [AuthUIActions Implementation](https://github.com/etn-ccis/blui-react-workflows/blob/master/login-workflow/example/src/actions/AuthUIActions.tsx)
+
+## Update RegistrationUIActions
+
+Update type name `AccountDetailInformation` to `AccountDetails`.
+
+// before
+```tsx
+import { RegistrationUIActions, AccountDetailInformation } from '@brightlayer-ui/react-auth-workflow';
+```
+
+// after
+```tsx
+import { RegistrationUIActions, AccountDetails } from '@brightlayer-ui/react-auth-workflow';
+```
+
+### Fix Following Actions in RegistrationUIActions.tsx
+
+#### Rename `loadEULA` action to `loadEula`.
+// before
+```tsx
+loadEULA: async (language: string): Promise<string> => {
+    await sleep(1000);
+
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+
+    if (!language.includes('en')) {
+        return 'Other language EULA';
+    }
+
+    return SAMPLE_EULA;
+},
+```
+
+// after
+```tsx
+loadEula: async (language: string): Promise<string> => {
+    await sleep(1000);
+
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+
+    if (!language.includes('en')) {
+        return 'Other language EULA';
+    }
+
+    return SAMPLE_EULA;
+}
+```
+
+#### Update completeRegistration Action
+
+// Before
+```tsx
+completeRegistration: async (
+    userData: {
+        password: string;
+        accountDetails: AccountDetailInformation;
+    },
+    validationCode: string,
+    validationEmail?: string
+): Promise<{ email: string; organizationName: string }> => {
+    const email = 'example@email.com';
+    const organizationName = 'Acme Co.';
+    const userInfo = { email, organizationName };
+
+    await sleep(1000);
+    if (isRandomFailure()) {
+        throw new Error('Sorry, there was a problem sending your request.');
+    }
+    return userInfo;
+},
+```
```

Comment:
It's a little difficult to see the difference here at a quick glance. Can you add more information on what the change is before showing the code example.